app/src/main/java/fr/free/nrw/commons/media/ZoomableActivity.kt

```diff
@@ -162,15 +152,6 @@ class ZoomableActivity : BaseActivity() {
         viewModel.result.observe(this) {
             handleResult(it)
         }
-
-        if(prefs.getBoolean(CustomSelectorConstants.FULL_SCREEN_MODE_FIRST_LUNCH, true)) {
-            // show welcome dialog on first launch
-            showWelcomeDialog()
```

Comment:
Unfortunately this prevents the first-time popup from appearing where it is meant to appear.

This first-time-popup is meant to only appear when using full-screen mode inside the **custom selector** (in the main activity, when you push the +, the custom selector is the third button at the bottom), not when using full-screen inside Explore or other places.

Reply:
Fixed, By Passing Origin in intent